srcpkgs/python3-scipy/template

```diff
@@ -1,25 +1,52 @@
 # Template file for 'python3-scipy'
 pkgname=python3-scipy
-version=1.3.2
-revision=2
+version=1.5.0
+revision=1
 wrksrc="scipy-${version}"
 build_style=python3-module
-pycompile_module="scipy"
 make_check_args="--force"
-hostmakedepends="python3-setuptools python3-Cython gcc-fortran"
-makedepends="python3-devel python3-numpy lapack-devel"
+hostmakedepends="python3-setuptools python3-Cython python3-numpy gcc-fortran"
+makedepends="python3-devel python3-numpy
+ lapack-devel openblas-devel python3-pybind11"
```

Comment:
What's up with the indenting here?

Reply:
The was getting long. But it fits on my terminal, so I joined it.